packages/react/src/drafts/UnderlinePanels/UnderlinePanels.features.stories.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import type {Meta} from '@storybook/react'
+import {INITIAL_VIEWPORTS} from '@storybook/addon-viewport'
+import UnderlinePanels from './UnderlinePanels'
+import type {ComponentProps} from '../../utils/types'
+import {
+  CodeIcon,
+  CommentDiscussionIcon,
+  EyeIcon,
+  GearIcon,
+  GitPullRequestIcon,
+  GraphIcon,
+  PlayIcon,
+  ProjectIcon,
+  ShieldLockIcon,
+} from '@primer/octicons-react'
+
+export default {
+  title: 'Drafts/Components/UnderlinePanels/Features',
+  component: UnderlinePanels,
+} as Meta<ComponentProps<typeof UnderlinePanels>>
+
+export const SelectedTab = () => (
+  <UnderlinePanels aria-label="Select a tab" id="tab-panels">
+    <UnderlinePanels.Tab>Tab 1</UnderlinePanels.Tab>
+    <UnderlinePanels.Tab aria-selected={true}>Tab 2</UnderlinePanels.Tab>
+    <UnderlinePanels.Tab>Tab 3</UnderlinePanels.Tab>
+    <UnderlinePanels.Panel>Panel 1</UnderlinePanels.Panel>
+    <UnderlinePanels.Panel>Panel 2</UnderlinePanels.Panel>
+    <UnderlinePanels.Panel>Panel 3</UnderlinePanels.Panel>
+  </UnderlinePanels>
+)
+
+export const LabelledByExternalElement = () => (
+  <>
+    <h2 id="my-heading">UnderlinePanels example</h2>
+    <UnderlinePanels aria-labelledby="my-heading">
+      <UnderlinePanels.Tab>Tab 1</UnderlinePanels.Tab>
+      <UnderlinePanels.Tab>Tab 2</UnderlinePanels.Tab>
+      <UnderlinePanels.Tab>Tab 3</UnderlinePanels.Tab>
+      <UnderlinePanels.Panel>Panel 1</UnderlinePanels.Panel>
+      <UnderlinePanels.Panel>Panel 2</UnderlinePanels.Panel>
+      <UnderlinePanels.Panel>Panel 3</UnderlinePanels.Panel>
+    </UnderlinePanels>
+  </>
+)
+
+export const WithIcons = () => (
+  <UnderlinePanels aria-label="Tabs with icons">
+    <UnderlinePanels.Tab icon={CodeIcon}>Code</UnderlinePanels.Tab>
```

Comment:
This is nitpick but how about using tab panel examples from dotcom or just tab1, tab2 etc? Because Code, Issues etc are navigational items and I feel we might mislead folks by using our core navigational usages for tab panels examples. Let me know what you think!